src/IB/IBInstrumentPanel.cpp

```diff
@@ -1233,7 +1233,7 @@ IBInstrumentPanel::writePlotData(const int timestep_num, const double simulation
     // Output the web data on the available MPI processes.
     for (unsigned int meter = 0; meter < d_num_meters; ++meter)
     {
-        if (meter % mpi_nodes == mpi_rank)
+        if (int(meter) % mpi_nodes == mpi_rank)
```

Comment:
Do we have a guideline about when to use a construction like `int(meter)` vs. `static_cast<int>(meter)`? I typically use `static_cast` to make it easier to make it easy to find these kinds of things later on using simple search expressions.

Reply:
Lets stick with `static_cast` - they do the same thing but that is the more common style in the codebase.